hack/kind/setup.sh

```diff
@@ -37,7 +37,7 @@ kind create cluster \
   --image="kindest/$kind_node_image" || true
 
 # shellcheck disable=SC2034
-KUBECONFIG="$(kind get kubeconfig-path --name="test-infra")"
+export KUBECONFIG="$(kind get kubeconfig-path --name="test-infra")"
```

Comment:
I believe that unless this script is sourced, then the export won't get set outside the script. Do you think we should recommend sourcing it, or just output a recommendation on how to set the kubeconfig?

Reply:
That's correct; I used `export` here to make the kubeconfig work within the script, so that it wouldn't get hung here. But I think you're right that we should recommend sourcing it the same way at the end of the script. I just updated the ending message to be more direct about that.